packages/main/tests/python/test_robocorp_workitems.py

```diff
@@ -714,14 +714,37 @@ def test_parse_work_item_from_email(self, library, email_file, expected_body):
         library.adapter.DATA["workitem-id-first"]["rawEmail"] = raw_email
 
         library.get_input_work_item()
-        body = library.get_work_item_variable("parsedEmail")["Body"]
-        assert body == expected_body
+        parsed_email = library.get_work_item_variable("parsedEmail")
+        assert parsed_email["Body"] == expected_body
 
     def test_parse_work_item_from_email_missing_content(self, library):
         library.get_input_work_item()
         with pytest.raises(KeyError):
             library.get_work_item_variable("parsedEmail")
 
+    @pytest.mark.parametrize(
+        "email_var, is_file, body",
+        [
+            ("email.text", False, "A message from e-mail"),
+            ("__mail.html", True, "from email"),
+        ],
+    )
+    def test_parse_work_item_from_parsed_email(self, library, email_var, is_file, body):
+        if is_file:
+            email_content = (RESOURCES_DIR / "work-items" / email_var).read_bytes()
```

Comment:
Could it be an idea to have the files delivered as fixtures to the tests?

Reply:
Yes, that's a good idea, actually I needed two different fixtures, one for the raw e-mail data and one for the parsed one. Here's the [change](https://github.com/robocorp/rpaframework/pull/487/commits/a8a2212f3e238d932c85acd9dfb779665a6ef14b#diff-1a9fb0149d775661889056b303db7b0bf250f510eb3b6bf10448bbbf805543e2R164-R190), let me know through an example if you want to improve it more. Thx!